src/l500/l500-private.cpp

```diff
@@ -163,5 +164,35 @@ namespace librealsense
             auto res = _hwm.send( cmd );
             _record_action( *this );
         }
+
+        void freefall_option::enable( bool e )
+        {
+            if( e != is_enabled() )
+            {
+                if( !e  &&  is_true() )
+                    set( 0 );
+                _enabled = e;
+            }
+        }
+
+        hw_sync_option::hw_sync_option( hw_monitor& hwm, std::shared_ptr< freefall_option > freefall_opt )
+            : bool_option( false )
+            , _hwm( hwm )
+            , _freefall_opt( freefall_opt )
```

Comment:
The logic requires that freefall will be allocated first. 
To enforce it I propose to intentionally throw an exception with  appropriate message in this ctor.
In case this scenario is valid (i.e. Freefall disabled in FW)  then just add documentation in l500 ctor